test/gui/shared/steps/steps.py

```diff
@@ -392,3 +395,22 @@ def step(context, resource, password):
     mouseClick(waitForObject(names.oCC_ShareLinkWidget_lineEdit_password_QLineEdit), 0, 0, Qt.NoModifier, Qt.LeftButton)
     type(waitForObject(names.oCC_ShareLinkWidget_lineEdit_password_QLineEdit), password)
     clickButton(waitForObject(names.oCC_ShareLinkWidget_createShareButton_QPushButton))
+
+
+@When('the user edits the public link named "|any|" of file "|any|" changing following')
+def step(context, publicLinkName, resource):
+    snooze(5)
+    test.compare(str(waitForObjectExists(names.sharingDialog_label_name_QLabel).text), resource)
+    test.compare(str(waitForObjectExists(names.linkShares_0_0_QModelIndex).text), publicLinkName)
+    expDate = []
+    for row in context.table:
+        if row[0] == 'expireDate':
+            expDate = row[1].split("-")
+
+    doubleClick(waitForObject(names.oCC_ShareLinkWidget_qt_spinbox_lineedit_QLineEdit), 5, 9, Qt.NoModifier, Qt.LeftButton)
```

Comment:
this doubleclick seems not to be needed

Reply:
The `doubleClick` is needed because we can only update the expiration date value by double clicking and selecting the current date and then typing the new value to replace the old value of dates.

Reply:
hmm when I try it manually I even need to triple click to select the complete date

Reply:
need a way to set the env. variable `LC_TIME=en_US.utf8` before starting the app under test